Automattic/wp-calypso
# Social Previews: Add Tumblr link preview

Related to 1204250791794927-as-1204281808721020

## Proposed Changes

This PR implements the Tumblr post preview.

## Testing Instructions

### Prerequisites
- Make sure you have a Tumblr account.
- Make sure you have a self-hosted site connected to Jetpack, with a subscription that includes Social (e.g., Jetpack Complete).
- From the _Social_ or _My Jetpack_ page of your site WPadmin, activate Social.
- Then, connect your site to Tumblr (check [these instructions](https://jetpack.com/support/jetpack-social/connecting-to-social-networks/) if you need help).
- Spin up Calypso, by running this branch locally or using a live link below.

### Testing

- Create a new post from your site WPadmin
- In the Jetpack sidebar, make sure the post will be published automatically to your Tumblr page
<img width="395" alt="Screenshot 2023-04-11 at 4 43 17 PM" src="https://user-images.githubusercontent.com/1620183/231283940-732fc281-184d-40f5-9415-22ef3c7c8e53.png">

- Publish it
- Visit `/posts/:site` in Calypso
- You should see the newly created post under the _Published_ tab
- In the post menu, click _Share_, then press the _Preview_ button
<img width="500" alt="Screenshot 2023-04-11 at 4 47 11 PM" src="https://user-images.githubusercontent.com/1620183/231284467-7d3c5008-321b-444f-a3f9-8b732eaa323c.png">

- Select the Tumblr preview
- You'll see a preview of how the post looks when shared



<img width="500" alt="" src="https://user-images.githubusercontent.com/1620183/234973251-d1b25539-3d70-4f73-a0eb-558ebc2e48c9.png">
<img width="500" alt="" src="https://user-images.githubusercontent.com/1620183/234973249-f53f2734-ce32-4caa-8ea4-2ab392796993.png">

## Pre-merge Checklist
